README.md

```diff
@@ -155,6 +155,22 @@ require "super_diff/rspec"
 
 ## Configuration
 
+If you want to use something other than the default colors, you can
```

Comment:
The Configuration section feels like it needs two subsections now. What do you think about adding a subheader here like `### Configuring colors`?